Panic caused by MouseClickNode function in chromedp/input.go

<!---
This issue tracker is mainly for bugs and feature requests. Before asking a
question, search online and try to investigate on your own.
-->

#### What versions are you running?

<pre>
$ go list -m github.com/chromedp/chromedp
github.com/chromedp/chromedp v0.5.4-0.20200827070026-4382e54a76d2

$ google-chrome --version
Version 85.0.4183.102 (Official Build) (64-bit)

$ go version
go version go1.13.8 windows/amd64
</pre>


In the file input.go, under MouseClickNode function, 'boxes' can have 0 length causing panic.
````
	boxes, err := dom.GetContentQuads().WithNodeID(n.NodeID).Do(ctx)
	if err != nil {
		return err
	}
	content := boxes[0]
````

````
panic: runtime error: index out of range [0] with length 0

goroutine 14461 [running]:
github.com/chromedp/chromedp.MouseClickNode.func1(0xf130c0, 0xc0002ba810, 0x0, 0x0)
	E:/Projects/go/pkg/mod/github.com/chromedp/chromedp@v0.5.4-0.20200827070026-4382e54a76d2/input.go:78 +0x7a7
github.com/chromedp/chromedp.ActionFunc.Do(0xc0007ce6f0, 0xf130c0, 0xc0002ba810, 0x0, 0x0)
	E:/Projects/go/pkg/mod/github.com/chromedp/chromedp@v0.5.4-0.20200827070026-4382e54a76d2/chromedp.go:607 +0x49
github.com/chromedp/chromedp.Click.func1(0xf130c0, 0xc0002ba810, 0x2a, 0xc000066c70, 0x2, 0x2, 0x0, 0x0)
	E:/Projects/go/pkg/mod/github.com/chromedp/chromedp@v0.5.4-0.20200827070026-4382e54a76d2/query.go:1003 +0x1a7
github.com/chromedp/chromedp.(*Selector).Do(0xc000a9a700, 0xf130c0, 0xc0002ba810, 0x0, 0x0)
	E:/Projects/go/pkg/mod/github.com/chromedp/chromedp@v0.5.4-0.20200827070026-4382e54a76d2/query.go:221 +0x72e
github.com/chromedp/chromedp.Tasks.Do(0xc00078a4c0, 0x2, 0x2, 0xf130c0, 0xc0002ba810, 0x0, 0x0)
	E:/Projects/go/pkg/mod/github.com/chromedp/chromedp@v0.5.4-0.20200827070026-4382e54a76d2/chromedp.go:617 +0xe3
github.com/chromedp/chromedp.Tasks.Do(0xc0009a3148, 0x1, 0x1, 0xf130c0, 0xc0002ba810, 0x0, 0x0)
	E:/Projects/go/pkg/mod/github.com/chromedp/chromedp@v0.5.4-0.20200827070026-4382e54a76d2/chromedp.go:617 +0xe3
github.com/chromedp/chromedp.Run(0xf13080, 0xc000862360, 0xc0009a3148, 0x1, 0x1, 0x0, 0x0)
	E:/Projects/go/pkg/mod/github.com/chromedp/chromedp@v0.5.4-0.20200827070026-4382e54a76d2/chromedp.go:275 +0x199
````

Suggested solution is to check if length of 'boxes' is > 0 before continuing:
````
	boxes, err := dom.GetContentQuads().WithNodeID(n.NodeID).Do(ctx)
	if err != nil {
		return err
	}

	if len(boxes) < 1 {
		return ErrInvalidDimensions
	}

	content := boxes[0]
````

--- a/input.go
+++ b/input.go
@@ -69,6 +69,11 @@ func MouseClickNode(n *cdp.Node, opts ...MouseOption) MouseAction {
 		if err != nil {
 			return err
 		}
+
+		if len(boxes) == 0 {
+			return ErrInvalidDimensions
+		}
+
 		content := boxes[0]
 
 		c := len(content)

